__tests__/app.test.js

```diff
@@ -15,3 +16,14 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects, each with a slug and a description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toEqual(topicsJson);
+      });
+  });
+});
```

Comment:
Now that we have set up a few endpoints, it would be good to implement a catch-all type of handler that sends a custom 404 if a user tries to request an endpoint that doesn't exist in our app